packages/site/src/components/AtlantisPreviewEditorProvider.tsx

```diff
@@ -0,0 +1,330 @@
+import { transform } from "@babel/standalone";
```

Comment:
This file is a bit of a mess. I tend to circle back and tidy it up a bit. It was in the books to do today, but I think it's more important to get this over the fence to get the new format unlocked versus tidying up some messy code.